cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+  },
+});
```

Comment:
Since you're not doing anything inside `setupNodeEvents`, I think you can remove it (which leaves `e2e` empty, so can probably remove it too).
```suggestion
module.exports = defineConfig({});
```

Reply:
I guess this probably came with some kind of cypress init command.